### PR TITLE
feat: Implement active speakers

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/components/calling/ParticipantMicOnIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+const ParticipantMicOnIcon: React.FC<ParticipantMicOnIconProps> = ({
+  className,
+  isActive = false,
+  activeColor = '#fff',
+  ...props
+}) => {
+  return (
+    <span
+      css={css`
```
Comment (Yserz) [inline src/script/components/calling/ParticipantMicOnIcon.tsx]: No need to call extra `css` here. `css={{...}}` would be enough.

File: src/script/calling/CallingRepository.ts [hunk 2]
```diff
@@ -1193,6 +1194,20 @@ export class CallingRepository {
     return this.mediaStreamQuery;
   };
 
+  private readonly updateActiveSpeakers = (wuser: number, conversationId: string, rawJson: string) => {
+    const call = this.findCall(conversationId);
+    if (!call) {
+      return;
+    }
+
+    const activeSpeakers = JSON.parse(rawJson);
```
Comment (Yserz) [inline src/script/calling/CallingRepository.ts:1203]: `rawJson` is guaranteed to be at least `{"audio_levels": []}`, right?

File: src/script/components/calling/ParticipantMicOnIcon.tsx (hunk 1 above)
Comment (thisisamir98) [reply src/script/components/calling/ParticipantMicOnIcon.tsx]: nice catch, https://github.com/wireapp/wire-webapp/pull/10094/commits/a4d957a9133cd531722a0027e338dc1060c57599

**Review by Yserz — APPROVED** [review]

File: src/script/calling/CallingRepository.ts (hunk 2 above)
Comment (thisisamir98) [reply src/script/calling/CallingRepository.ts:1203]: avs said it's fine

**Pull request merged**